### PR TITLE
Add conflicts with other packages that provide the same hook

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
/usr/lib/build/kiwi_post_run

Signed-off-by: Jordi Massaguer Pla <jmassaguerpla@suse.de>

### CODE REVIEW

File: packaging/suse/make_spec.sh [hunk 1]
```diff
@@ -59,6 +59,8 @@ Requires:       rubygem-changelog_generator
 %endif
 Requires:       changelog-generator-data
 Requires:       libxml2-tools
+# Conflicts with other packages that provide /usr/lib/build/kiwi_post_run
+Conflicts:      infos-creator-rpm
```
Comment (davidcassany) [inline packaging/suse/make_spec.sh:63]: @jordimassaguerpla just out of curiosity, do you know how infos-creator-rpm is used or what feature provides?

**Review by davidcassany — APPROVED** [review]
LGTM

File: packaging/suse/make_spec.sh (hunk 1 above)
Comment (jordimassaguerpla) [reply packaging/suse/make_spec.sh:63]: no idea

**Review by MaximilianMeister — APPROVED** [review]
lgtm

**Pull request merged**